webpack.prod.js

```diff
@@ -1,16 +1,17 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] Same here, it is always a good practice to make sure you fix all linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉